stress-example/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# seems that BiT only works in 2.0.0 not 2.2.0, but hey it's google's problem!
+tensorflow_cpu==2.1.0
```

Comment:
So then should this be `2.0.0`?

Reply:
we should use ImageTorchEncoder at the end